ci_framework/roles/ci_make/molecule/default/molecule.yml

```diff
@@ -0,0 +1,28 @@
+---
+driver:
+  name: podman
+
```

Comment:
We need to add platforms here:
`platforms:
  - name: centos
    hostname: centos
    pre_build_image: true
    image: ${TRIPLEO_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
    registry:
      url: ${TRIPLEO_ANSIBLE_MOLECULE_REGISTRY:-"registry.access.redhat.com"}`